gammapy/cube/tests/test_fit.py

```diff
@@ -62,6 +62,13 @@ def exposure(geom):
                                                  offset_max=offset_max)
     return exposure_map
 
+@pytest.fixture(scope='session')
+def background(geom):
+    m = Map.from_geom(geom)
+    m.quantity = np.ones((9, 150, 400))*1e-5
```

Comment:
Can you infer the data shape from the `geom` object, when `geom` changes it will fail otherwise...